contrib/utilities/indent

```diff
@@ -31,49 +46,49 @@ if test ! -f contrib/utilities/astyle.rc ; then
   exit 1
 fi
 
-if test -z "`which astyle`" ; then
+astyle="$(find_astyle)"
+astyle="${astyle:-`which astyle`}" # Fallback to `which astyle`
+
+if test -z "${astyle}"; then
   echo "*** No astyle program found."
   exit 1
 fi
 
-if test "`astyle --version 2>&1`" != "Artistic Style Version 2.04" ; then
+if [[ "$("${astyle}" --version 2>&1)" != "Artistic Style Version 2.04" ]]; then
```

Comment:
is `[[` any different from `[`?

Reply:
Yes, [here](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-_005b_005b) and [here](https://unix.stackexchange.com/a/255483) have some info on the difference. `[[` behaves more predictably than `[` in a lot of cases, but I think it's not POSIX compliant. I used it here because for some reason zsh doesn't like '==' as a comparison operator in an ordinary `[` environment. As above though, not necessary if there's no need to support zsh.

Reply:
Let's stick with `test` then -- that's what's used everywhere else.